test/e2e/manifest/containerd-windows.json

```diff
@@ -31,7 +31,7 @@
         "windowsProfile": {
             "adminUsername": "azureuser",
             "adminPassword": "replacepassword1234$",
-            "csiProxyURL": "https://acs-mirror.azureedge.net/csi-proxy/v1.1.1/binaries/csi-proxy-v1.1.1.tar.gz",
+            "csiProxyURL": "https://vitaliyorgstorage.azureedge.net/9d4196ab-1c3c-4efb-b065-b643384f1832/github/csi-proxy-v1.1.2-20220808-1700.tar.gz",
```

Comment:
it won't work, the json file need to be merged into master branch first, and the e2e test would be run against modified csi-proxy address.

Reply:
ok. I will rollback this files. 
Now I want to pass 'Windows Unit Tests' with new csi-proxy. It works on my on-premise cluster, but I want to be sure that tests are correct.

Reply:
Windows Unit Tests with new csi-proxy have been passed.

Reply:
csi-proxy has been restored to v1.1.1